sdk/requirements.dev.txt

```diff
@@ -1 +1,2 @@
-parameterized==0.7.*
\ No newline at end of file
+parameterized==0.7.*
+lusidfeature >= 0.1.3
```

Comment:
remove lusidfeature from here and add the following as the first line

`-r requirements.txt`